### ISSUE TITLE
pyembed-markdown no longer working with Pelican and YouTube?

### ISSUE THREAD

**Issue by AWegnerGitHub** [pyembed/pyembed-markdown#24]
I am currently using pyembed-markdown 1.1.0

I am having problems generating my Pelican pages - old pages that I haven't touched in years - if they have a YouTube video embedded. 

When generating Pelican debug pages via:

`pelican content --debug --autoreload  --output output --settings pelicanconf.py`

I get the following traceback for a page that has a YouTube embed:

```
DEBUG: Read file 2011_02_04_a-new-more-fair-rtd.md-> Article -> Starting new HTTP connection (1): oembed.com
DEBUG: "GET /providers.json HTTP/1.1" 200 3286-> Starting new HTTPS connection (1): www.youtube.com
DEBUG: "GET /watch?v=OzHNr1Bz5QQ HTTP/1.1" 302 359-> Starting new HTTPS connection (1): ipv4.google.com
DEBUG: "GET /sorry/IndexRedirect?continue=https://www.youtube.com/watch%3Fv%3DOzHNr1Bz5QQ&q=CGMSBMfnSUQYx6nFuAUiGQDxp4NLsAf7w-2Mq_88UJmpQioznStS3bk HTTP/1.1" 503 2883
ERROR: Could not process .\2011_02_04_a-new-more-fair-rtd.md
  | PyEmbedConsumerError: No valid OEmbed responses for URLs <generator object get_oembed_urls at 0x00000000058D9C18>
  |___
  | Traceback (most recent call last):
  |   File "h:\Anaconda\lib\site-packages\pelican\generators.py", line 514, in generate_context
  |     context_sender=self)
  |   File "h:\Anaconda\lib\site-packages\pelican\readers.py", line 508, in read_file
  |     content, reader_metadata = reader.read(path)
  |   File "h:\Anaconda\lib\site-packages\pelican\readers.py", line 287, in read
  |     content = self._md.convert(text)
  |   File "h:\Anaconda\lib\site-packages\markdown\__init__.py", line 375, in convert
  |     newRoot = treeprocessor.run(root)
  |   File "h:\Anaconda\lib\site-packages\markdown\treeprocessors.py", line 295, in run
  |     self.__handleInline(text), child
  |   File "h:\Anaconda\lib\site-packages\markdown\treeprocessors.py", line 106, in __handleInline
  |     data, patternIndex, startIndex)
  |   File "h:\Anaconda\lib\site-packages\markdown\treeprocessors.py", line 239, in __applyPattern
  |     node = pattern.handleMatch(match)
  |   File "h:\Anaconda\lib\site-packages\pyembed\markdown\pattern.py", line 44, in handleMatch
  |     html = self.pyembed.embed(url, max_width, max_height)
  |   File "h:\Anaconda\lib\site-packages\pyembed\core\__init__.py", line 59, in embed
  |     oembed_urls, max_width=max_width, max_height=max_height)
  |   File "h:\Anaconda\lib\site-packages\pyembed\core\consumer.py", line 57, in get_first_oembed_response
  |     raise PyEmbedConsumerError('No valid OEmbed responses for URLs %s' % oembed_urls)
  | PyEmbedConsumerError: No valid OEmbed responses for URLs <generator object get_oembed_urls at 0x00000000058D9C18>
```

My environment has not changed and I create new posts every couple months so I don't have an exact timeframe of when this began. 

How can I continue to embed YouTube videos in my posts?

**Comment by matt-thomson** [comment]
It looks like YouTube are clamping down on requests to their video pages from automated scripts.  I've just released pyembed v1.3.3, which adds an explicit override for YouTube.  Let me know if this helps!

**Comment by AWegnerGitHub** [comment]
Thank you @matt-thomson . That seems to have solved the issue.
